frontend/src/components/SideNav.tsx

```diff
@@ -80,6 +81,12 @@ export const SideNav = ({ activePage }) => {
         newTab
         active={false}
       />
+      <SideNavItem
+        icon={UserCircleIcon}
+        text="Professors"
```

Comment:
we should probably change the term "Professors" to "Instructors"
not all course instructors are necessarily professors